package/boot/uboot-rockchip/patches/U-Boot-v4-1-5-rockchip-Avoid-using-libfdt-with-of-platdata.diff

```diff
@@ -0,0 +1,65 @@
+diff --git a/drivers/clk/rockchip/clk_rk3328.c b/drivers/clk/rockchip/clk_rk3328.c
+index a89e2ecc4a..e86c17e6d6 100644
+--- a/drivers/clk/rockchip/clk_rk3328.c
++++ b/drivers/clk/rockchip/clk_rk3328.c
```

Comment:
Please properly name (and you prefixes for) these patch files as done for other packages.